crowbar_framework/app/models/node_object.rb

```diff
@@ -830,15 +830,29 @@ def destroy
   end
 
   def networks
-    self.crowbar["crowbar"]["network"] rescue {}
+    networks = {}
+    crowbar["crowbar"]["network"].each do |name, data|
+      # note that node might not be part of network proposal yet (for instance:
+      # if discovered, and IP got allocated by user)
```

Comment:
@vuntz I might be missing something here, but if the node is not part of the network proposal yet, I think we'd not even get here. Because ```crowbar["crowbar"]["network"]``` should also be empty in that case.